modules/appeals_api/config/schemas/v2/10182.json

```diff
@@ -0,0 +1,150 @@
+    },
+
+    "date": { "type": "string", "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$", "maxLength": 10, "minLength": 10 },
+
+    "nodCreateRoot": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "data": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "type": { "type": "string", "enum": ["noticeOfDisagreement"] },
+            "attributes": {
+              "type": "object",
+              "additionalProperties": false,
+              "properties": {
+                "veteran": {
+                  "type": "object",
+                  "additionalProperties": false,
+                  "properties": {
+                    "homeless": { "type": "boolean" },
+                    "address": {
+                      "type": "object",
+                      "additionalProperties": false,
+                      "properties": {
+                        "addressLine1": { "type": "string" },
+                        "addressLine2": { "type": "string" },
+                        "addressLine3": { "type": "string" },
+                        "city": { "type": "string" },
+                        "stateCode": {
+                          "type": "string",
+                          "enum": [
+                            "AL", "AK", "AS", "AZ", "AR", "AA", "AE", "AP", "CA", "CO",
+                            "CT", "DE", "DC", "FM", "FL", "GA", "GU", "HI", "ID", "IL",
+                            "IN", "IA", "KS", "KY", "LA", "ME", "MH", "MD", "MA", "MI",
+                            "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY",
+                            "NC", "ND", "MP", "OH", "OK", "OR", "PW", "PA", "PR", "RI",
+                            "SC", "SD", "TN", "TX", "UT", "VT", "VI", "VA", "WA", "WV",
+                            "WI", "WY"
+                          ]
+                        },
+                        "countryName": { "type": "string" },
+                        "zipCode5": {
+                          "type": "string",
+                          "pattern": "^[0-9]{5}$",
+                          "minLength": 5, "maxLength": 5,
+                          "description": "5-digit zipcode. Use \"00000\" if Veteran is outside the United States"
+                        },
+                        "internationalPostalCode": { "type": "string" }
+                      },
+                      "required": [ "addressLine1", "city", "countryName", "zipCode5" ]
+                    },
+                    "phone": {
+                      "$comment": "the phone fields must not exceed 20 chars, when concatenated",
```

Comment:
Is this still true? Does the new form allow us to get away with longer phone numbers?

Reply:
It looks like it allows for `24` chars without any formatting. Do you think we should we change this or leave this as is?

Reply:
Eh, close enough to 20 to leave as-is.